src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -98,7 +98,12 @@ struct CPUStreamsExecutor::Impl {
                     // wrapping around total_streams (i.e. how many streams all different core types can handle
                     // together)
                     const auto total_streams = _impl->total_streams_on_core_types.back().second;
+                    const auto core_type_size = _impl->total_streams_on_core_types.size();
                     const auto streamId_wrapped = _streamId % total_streams;
+                    const auto use_logic = _impl->_config._threads_per_stream_big * _impl->_config._big_core_streams >
```

Comment:
Could you descript its details?

Reply:
`use_logic` means to use logic core?

Reply:
Yes, current stream is placed on logical core.